src/generators/css-filter.ts

```diff
@@ -63,7 +76,10 @@ export function cssFilterStyleheetTemplate(filterValue: string, reverseFilterVal
     });
 
     if (!frameURL) {
-        const [r, g, b] = applyColorMatrix([255, 255, 255], createFilterMatrix(config));
+        let [r, g, b] = applyColorMatrix([255, 255, 255], createFilterMatrix(config));
```

Comment:
`createFilterMatrix(config)` is called twice. `applyColorMatrix` is also called twice for users of Chrome >=81 (who are now majority). So maybe let's use `if` ... `else`.